api/py/ai/chronon/repo/compile.py

```diff
@@ -106,6 +106,12 @@ def extract_and_convert(chronon_root, input_path, output_root, debug, force_over
         if _write_obj(full_output_root, validator, name, obj, log_level, force_overwrite, force_overwrite):
             num_written_objs += 1
 
+            if obj_class is Join:
+                _print_highlighted("Output Join Features", get_join_output_columns(obj))
```

Comment:
I am thinking that we should use a different color for the names of the features - and have them be separated by new lines - especially for the join, the list can get really long.

Reply:
Hi @nikhilsimha do you mean we print out one feature per line? Will it makes the output too long to read since the features list can be really long?